src/ui/src/composables/useApplicationCloud.ts

```diff
@@ -31,16 +39,11 @@ export function useApplicationCloud(wf: Core) {
 	});
 	const appId = computed(() => wf.writerApplication.value?.id);
 
-	const deployUrl = computed(() => {
-		return new URL(
-			`/aistudio/organization/${orgId.value}/agent/${appId.value}/deploy`,
-			apiBaseUrl,
-		);
-	});
-
 	const liveUrl = computed(() => {
-		return deploymentInformation.value.applicationVersionData.data
-			?.deploymentUrl;
+		const teamId = deploymentInformation.value.writer?.teamIds?.[0];
```

Comment:
Need to fall back to  `cloud` team if app is deployed on cloud (`deploymentInformation.value.cloud?.teamIds?.[0]`). Discussion is in progress about that